src/integrations/watchers/indexable-post-watcher.php

```diff
@@ -201,7 +201,7 @@ public function build_indexable( $post_id ) {
 			$post = $this->post->get_post( $post_id );
 
 			// Build links for this post.
-			if ( $post && $indexable && $post->post_status !== 'draft' ) {
+			if ( $post && $indexable && $post->post_status === 'publish' ) {
```

Comment:
Should this maybe be filterable? As plugins / users may [introduce their own post statuses](https://developer.wordpress.org/reference/functions/register_post_status/).

Reply:
This is inline with how the link indexing tool already works. It only indexes public posts.

Reply:
There is method `is_public_post_status` which allow filtering of "public post statues" (filter `wpseo_public_post_statuses`). Please use this method instead of hard-coding 'public'.

Reply:
@stodorovic Thanks for that!

Just moved that function to a more logical place so we can use it where we need it.